### PR TITLE
V1.4b bugfix#498 peiduo

### PR DESCRIPTION
**Pull request by ZhaoPeiduo**

### CODE REVIEW

File: src/main/java/seedu/programmer/model/student/LabNum.java [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
 import static java.util.Objects.requireNonNull;
+import static seedu.programmer.commons.util.AppUtil.checkArgument;
 
 /**
  * Represents a student's name in the ProgrammerError.
  */
 public class LabNum {
+    public static final String MESSAGE_CONSTRAINT =
+            " Lab number should be between 0 and 13 (inclusive)";
```
Comment (reddevil1313) [inline src/main/java/seedu/programmer/model/student/LabNum.java:11]: Repeating MESSAGE_CONSTRAINT. A similar message constraint already exists in Lab, don't need to create a new one. You can just refer to the previous one.

File: src/main/java/seedu/programmer/model/student/LabResult.java [hunk 2]
```diff
@@ -1,28 +1,40 @@
...
 
 import static java.util.Objects.requireNonNull;
+import static seedu.programmer.commons.util.AppUtil.checkArgument;
 
 /**
  * Represents a student's name in the ProgrammerError.
  */
 public class LabResult {
+    public static final String MESSAGE_CONSTRAINT =
+            " Lab result should be a non-negative number or a unmarked placeholder(-1)";
```
Comment (reddevil1313) [inline src/main/java/seedu/programmer/model/student/LabResult.java:11]: Similarly for this one, it should refer to the constraint in Lab class.

**Review by reddevil1313 — DISMISSED** [review]

File: src/main/java/seedu/programmer/model/student/LabNum.java (hunk 1 above)
Comment (ZhaoPeiduo) [reply src/main/java/seedu/programmer/model/student/LabNum.java]: Ok have changed in the latest commit!

File: src/main/java/seedu/programmer/model/student/LabResult.java (hunk 2 above)
Comment (ZhaoPeiduo) [reply src/main/java/seedu/programmer/model/student/LabResult.java]: Ok have changed in the latest commit!

**Review by erwinqxy — APPROVED** [review]
LGTM

**Pull request merged**